index.html

```diff
@@ -5,22 +5,25 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="style.css">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+
 </head>
 <body>   
 
     <header class="headr">
 
         <div class="logo">
-            <a href="#home" class="log">John</a>
-            <a href="#" class="menuBtn"><img src="unt/Union.png" alt=""></a>
+            <a href="#" class="log">John</a>
```

Comment:
- Kindly, It would be better to implement `hover` and `focus` state for all your links to be easily identified 👍👌 Your logo hasn't have any hover and focuses so it failing color contrast
![Screenshot_20230124_112732](https://user-images.githubusercontent.com/52010789/214402219-3565ad52-481d-4899-9167-a737e0c486ec.png)